cpp/src/arrow/compute/exec/hash_join.cc

```diff
@@ -604,8 +654,84 @@ class HashJoinBasicImpl : public HashJoinImpl {
     return Status::OK();
   }
 
+  Status ApplyBloomFiltersToBatch(size_t thread_index, ExecBatch& batch) {
+    if (batch.length == 0) return Status::OK();
+    int64_t bit_vector_bytes = bit_util::BytesForBits(batch.length);
+    std::vector<uint8_t> selected(bit_vector_bytes);
+    std::vector<uint32_t> hashes(batch.length);
+    std::vector<uint8_t> bv(bit_vector_bytes);
+
+    RETURN_NOT_OK(InitLocalStateIfNeeded(thread_index));
+    // Start with full selection for the current minibatch
```

Comment:
```suggestion
    // Start with full selection for the current batch
```
I'm pretty sure "minibatch" has a different meaning for when we split input up into size 1024 batches on the temp vector stack but I could be wrong.

Reply:
Yes, you're right